tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -43,7 +43,7 @@ internal static Scope SendStartInvocation(ILambdaExtensionRequest requestBuilder
         var request = requestBuilder.GetStartInvocationRequest();
         WriteRequestPayload(request, data);
         WriteRequestHeaders(request, context);
-        var response = (HttpWebResponse)request.GetResponse();
+        using var response = (HttpWebResponse)request.GetResponse();
```

Comment:
Would you mind updating this line as well as this as well, it seems to be the one other place where we are invoking this API and we're not disposing it there as well: https://github.com/DataDog/dd-trace-dotnet/blob/0252136ccf64f784c94a04ff0dcd756dd87f5d6a/tracer/src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs#L180

Reply:
done, thanks!

Reply:
oh, yeah. oops, I didn't even look.